src/firebase/firestore/job.ts

```diff
@@ -57,17 +61,22 @@ export const updateMonthlyCounter = async (
   await updateDoc(docRef, data);
 };
 
-export const createJob = async (job: Job): Promise<void> => {
+export const createJob = async (job: Partial<Job>): Promise<void> => {
   const docRef = collection(db, 'jobs');
-  const monthlyCounter = await getMonthlyCounter();
-  const now = new Date();
-  await updateMonthlyCounter(now, monthlyCounter + 1);
+  try {
+    const monthlyCounter = await getMonthlyCounter();
+    const now = new Date();
+
+    const jobId =
+      now.getFullYear().toString().slice(-2) +
+      now.getMonth().toString() +
+      (monthlyCounter + 1).toString();
```

Comment:
there's a bug here where the length of the job id changes depending on whether the  `monthlyCounter` is one or two digits i.e. when `monthlyCounter=9` the id is `22109` and when `monthlyCounter=10` the id is `221010`. To expedite progress for other ppl (specifically Annie) we'll let this go for this PR and address it in a separate one (will be added as a sprint task).

Reply:
fixed